# Scene graph for AutonomousRGBDCamera

ObjectDescription[] objects                  # A list of scene objects handled by AutonomousRGBDCamera
ObjectRelation[]    relations                # A list of scene object spatial relationships